.travis.yml

```diff
@@ -17,15 +15,10 @@ install:
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
   - unset VIRTUAL_ENV
-  # Python 3.4 and 2.7 need six installed before running setup.py
-  - pip install six
   # Travis needs to update pip/setuptools
   - pip install --upgrade setuptools pip
   - pip install .
   - |
```

Comment:
```suggestion
```

This is a marker for the block on the lines below.

Reply:
updated